### PR TITLE
Move most archives tests into jbosstools-server

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

**Comment by rawagner** [comment]
Where is the JIRA reference ? I havent heard about any plan to move archives tests to server repository.

I see that you already commited these tests to servers but left out two classes - we are going to stop testing deploying archives ? why ? 
You are also missing a lot of configuration in servers repo like https://github.com/jbosstools/jbosstools-integration-tests/blob/master/tests/pluginCustomization.ini and all these properties https://github.com/jbosstools/jbosstools-integration-tests/blob/master/tests/pom.xml#L15. You wont download devstudio when running from cmd. Have you tried running them from jenkins environment on all platforms (OSX,Linux, Windows) ? I does not really matter to me if these tests run locally if they dont run in jenkins.

I think this needs more thought/discussion and really create a plan before moving stuff

**Comment by mmalina** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by rawagner** [comment]
testPR

**Comment by rawagner** [comment]
Sorry for the delay. We need to make sure that moved tests can be run against pre-installed devstudio and thanks to this PR https://github.com/jbosstools/jbosstools-build/pull/240 we are able to do it now. Can you please resolve the conflicts ? I will merge it then

File: tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/ProjectTestsSuite.java [hunk 1]
```diff
@@ -36,34 +36,34 @@
 @SuiteClasses({
```
Comment (rawagner) [inline tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/ProjectTestsSuite.java]: i think this file is not supposed to be changed in this PR

**Comment by psrna** [comment]
closing. archives have been removed by this pr: https://github.com/jbosstools/jbosstools-integration-tests/pull/1729

**Pull request closed without merging**